src/availableSupportedCards.cpp

```diff
@@ -60,8 +62,22 @@ std::vector<CardInfo::ptr> availableSupportedCards()
             }
         }
 
+#ifdef _WIN32
+        // In Windows, also include CryptoAPI tokens.
+        // Initially, CryptoAPI tokens will be included only if there are
+        // no tier 1/2 cards present.
+        if (cards.empty()) {
+            auto msCryptoApiElectronicIDs = listMsCryptoApiElectronicIDs();
+            if (!msCryptoApiElectronicIDs.empty()) {
+                seenCard = true;
+                cards = msCryptoApiElectronicIDs;
+            }
```

Comment:
```suggestion
            cards = listMsCryptoApiElectronicIDs();
```
seenCard is already previously set true